core/server/master/src/main/java/alluxio/master/meta/MetaMasterClientServiceHandler.java

```diff
@@ -12,36 +12,30 @@
 package alluxio.master.meta;
 
 import alluxio.Configuration;
-import alluxio.Constants;
 import alluxio.PropertyKey;
-import alluxio.RpcUtils;
 import alluxio.RuntimeConstants;
+import alluxio.grpc.BackupPOptions;
+import alluxio.grpc.BackupPResponse;
+import alluxio.grpc.GetConfigReportPOptions;
+import alluxio.grpc.GetConfigReportPResponse;
+import alluxio.grpc.GetConfigurationPOptions;
+import alluxio.grpc.GetConfigurationPResponse;
+import alluxio.grpc.GetMasterInfoPOptions;
+import alluxio.grpc.GetMasterInfoPResponse;
+import alluxio.grpc.GetMetricsPOptions;
+import alluxio.grpc.GetMetricsPResponse;
+import alluxio.grpc.MasterInfo;
+import alluxio.grpc.MasterInfoField;
+import alluxio.grpc.MetaMasterClientServiceGrpc;
 import alluxio.metrics.MetricsSystem;
-import alluxio.thrift.AlluxioTException;
-import alluxio.thrift.BackupTOptions;
-import alluxio.thrift.BackupTResponse;
-import alluxio.thrift.GetConfigReportTOptions;
-import alluxio.thrift.GetConfigReportTResponse;
-import alluxio.thrift.GetConfigurationTOptions;
-import alluxio.thrift.GetConfigurationTResponse;
-import alluxio.thrift.GetMasterInfoTOptions;
-import alluxio.thrift.GetMasterInfoTResponse;
-import alluxio.thrift.GetMetricsTOptions;
-import alluxio.thrift.GetMetricsTResponse;
-import alluxio.thrift.GetServiceVersionTOptions;
-import alluxio.thrift.GetServiceVersionTResponse;
-import alluxio.thrift.MasterInfo;
-import alluxio.thrift.MasterInfoField;
-import alluxio.thrift.MetaMasterClientService;
+import alluxio.util.RpcUtilsNew;
```

Comment:
can we get rid of the old rpcutils now that you've removed all thrift?

Reply:
Being addressed as part of boot-up and security work.

Reply:
Currently they are not referenced at all.